doc/classes/AudioStreamPlayer.xml

```diff
@@ -22,6 +22,7 @@
 			<description>
 				Returns the position in the [AudioStream] of the latest sound, in seconds. Returns [code]0.0[/code] if no sounds are playing.
 				[b]Note:[/b] The position is not always accurate, as the [AudioServer] does not mix audio every processed frame. To get more accurate results, add [method AudioServer.get_time_since_last_mix] to the returned position.
+				[b]Note:[/b] The returned position is always [code]0.0[/code] if the [AudioStream] is [AudioStreamInteractive]. [AudioStreamInteractive] may have multiple clips with transitions between them. This means that the playback position loses meaning, especially during transitions.
```

Comment:
```suggestion
				[b]Note:[/b] The returned position is always [code]0.0[/code] if the [member stream] is of type [AudioStreamInteractive], since it can have multiple clips playing at once.
```

I think it can be a bit shorter while still getting the point across. + refers to the "stream" property now instead of the class. Feel free to tweak my suggestion further if you like :)

Reply:
You can also add `particularly during transitions.` at the end if wanted, but I didn't think it's necessary to understand this behavior. And the sentence would get a bit long :p

Reply:
Continuing with the above suggestion.
```suggestion
				[b]Note:[/b] This method always returns [code]0.0[/code] if the [member stream] is an [AudioStreamInteractive], since it can have multiple clips playing at once.
```
I do agree to keep verbosity at a minimum, especially because this method doesn't have a "strict" dependance on this class. That is to say that you don't need to know about AudioStreamInteractive to make use of this method.

Reply:
That makes very much sense! Thank you very much for the feedback. I have fixed it now :)